projects/plugins/protect/src/js/routes/scan/scanning-admin-section-hero.tsx

```diff
@@ -35,20 +34,22 @@ const ScanningAdminSectionHero: React.FC = () => {
 								/>
 							) }
 							<Text>
-								{ sprintf(
-									// translators: placeholder is the number of total vulnerabilities i.e. "22,000".
-									__(
-										'We are scanning for security threats from our more than %s listed vulnerabilities, powered by WPScan. This could take a minute or two.',
-										'jetpack-protect'
-									),
-									totalVulnerabilitiesFormatted
-								) }
+								{ hasPlan
+									? __(
+											'Jetpack is actively scanning your site files line-by-line to identify threats and vulnerabilities. This could take a minute or two.',
```

Comment:
To maintain consistency with the prior message:
```suggestion
											"Jetpack is actively scanning your site's files line-by-line to identify threats and vulnerabilities. This could take a minute or two.",
```